internal/convert_test.go

```diff
@@ -60,7 +60,29 @@ func TestGetDDL(t *testing.T) {
 			"a": ddl.ColumnDef{Name: "a", T: ddl.Type{Name: ddl.Int64}},
 		},
 		Pks: []ddl.IndexKey{ddl.IndexKey{Col: "a"}}}
-	ddl := conv.GetDDL(ddl.Config{})
+	conv.SpSchema["table3"] = ddl.CreateTable{
```

Comment:
I don't see how this change in convert_test.go is connected with the change in convert.go (which changes the order of printing interleaved tables, and doesn't do anything with foreign key constraints).
Maybe it is left-over from a previous version of convert?

Separately: we should probably have a test of the new GetDDL functionality i.e. something to test out printing of interleaved tables.

Reply:
Noted: as this function and tests are updated majorly as part of https://github.com/cloudspannerecosystem/harbourbridge/pull/101, i'll update this function for interleaved tables, once PR#101 is merged.

Reply:
Sounds good -- thanks!

Reply:
just a note, this change is still pending, as this is updated as part of #95 , i'll update this once that is merged

Reply:
This change is now implemented.